src/networkParams.ts

```diff
@@ -86,15 +86,11 @@ export class TransactionOptions {
     /**
      * Returns true if Guarded Transaction Option is set
      */
-    isGuardedTx(): boolean {
-        if ((this.value & TRANSACTION_OPTIONS_TX_GUARDED_MASK) == TRANSACTION_OPTIONS_TX_GUARDED) {
-            return true
-        }
-        return false
+    withGuardedOptions(): boolean {
```

Comment:
I would rename this function to `hasGuardedOptions`, since `withGuardedOptions` sounds like a "setter"

Reply:
changed to `hasGuardedOption`